src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Link } from 'react-router';
+import {
+  isEmpty as _isEmpty,
+} from 'underscore';
+import appConfig from '../../../../appConfig';
+
+const WorkDetail = (props) => {
+  if (!props.detail) {
+    return null;
+  }
+
+  const { item } = props.detail;
+  const ebookUrl = appConfig.ereader[process.env.APP_ENV];
+
+  const parseEntities = (entities) => {
+    return (
+      <ul>
+        {entities.map((entity, i) => (
+          <li key={i.toString()}>
+            <Link to={{ pathname: '/', query: { author: `${entity.name}` } }}>{entity.name}, {entity.role}</Link>
+          </li>
+        ))}
+      </ul>
+    );
+  };
+
+  const parseSubjects = (subjects) => {
+    return (
+      <ul>
+        {subjects.map((subject, i) => (
+          <li key={i.toString()}>
+            <Link to={{ pathname: '/', query: { subject: `${subject.subject}` } }}>{subject.subject}</Link>
+          </li>
+        ))}
+      </ul>
+    );
+  };
+
+  const parseEbooks = (ebooks) => {
+    return (
+      <div className="ebook-links">
+        {ebooks.map((ebook, i) => (
+          <span key={i.toString()}>
+            <Link to={{ pathname: `${ebookUrl}`, query: { url: `${ebook.url}` } }}>{ebook.epub_path.split('/').pop()}</Link><br />
+          </span>
+        ))}
+      </div>
+    );
+  };
+
+  const parseInstances = (instances) => {
+    return (
+      <table className="nypl-basic-table">
```

Comment:
This is missing some accessibility attributes but I'd say make that a ticket for later.